src/ocean/LibVersion.d

```diff
@@ -0,0 +1,9 @@
+module ocean.LibVersion;
+
+const has_features_2_6 = true;
```

Comment:
> anyone still at v2.5.6 or lower will have the check failing

We don't support non-latest point releases, such users are strictly obliged to upgrade.